storage/driver/error_test.go

```diff
@@ -1,13 +1,39 @@
 package driver
 
 import (
+	"errors"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
 func TestError(t *testing.T) {
-	err := PathNotFoundError{Path: "/tmp"}
-	got := err.Error()
-	assert.Equal(t, "Path not found: /tmp", got)
+	testCases := []struct {
+		path      string
+		err       error
+		wantError string
+	}{
+		{
+			"", nil,
+			"Path not found: , err: %!s(<nil>)",
+		},
+		{
+			"", errors.New("hello"),
+			"Path not found: , err: hello",
+		},
+		{
+			"/tmp", nil,
+			"Path not found: /tmp, err: %!s(<nil>)",
+		},
+		{
+			"/tmp", errors.New("hello"),
+			"Path not found: /tmp, err: hello",
```

Comment:
```suggestion
			"", 
			nil,
			"Path not found: , err: %!s(<nil>)",
		},
		{
			"", 
			errors.New("hello"),
			"Path not found: , err: hello",
		},
		{
			"/tmp", 
			nil,
			"Path not found: /tmp, err: %!s(<nil>)",
		},
		{
			"/tmp", 
			errors.New("hello"),
			"Path not found: /tmp, err: hello",
```
LOC(Line of Code)는 몇 줄 늘어나기는 하지만,
가독성을 위해서 각 인자 당 한 줄씩 배치하는 쪽으로 의견 드립니다.

Reply:
네 스타일 논의네요.
전체 길이가 짧을 경우에 한해서
// inputs
// wants
이렇게 두줄 형식으로 하고 있습니다.
이렇게 하는 게 가독성이 높은 경우가 많더라고요.